sklearn/preprocessing/_function_transformer.py

```diff
@@ -40,18 +41,45 @@ class FunctionTransformer(BaseEstimator, TransformerMixin):
         kwargs forwarded. If inverse_func is None, then inverse_func
         will be the identity function.
 
-    validate : bool, optional default=True
+    validate : bool or 'array-or-frame', optional default=True
         Indicate that the input X array should be checked before calling
-        func. If validate is false, there will be no input validation.
-        If it is true, then X will be converted to a 2-dimensional NumPy
-        array or sparse matrix. If this conversion is not possible or X
-        contains NaN or infinity, an exception is raised.
+        func. The possibilities are:
+
+        - If True, then X will be converted to a 2-dimensional NumPy array or
+          sparse matrix. If the conversion is not possible an exception is
+          raised.
```

Comment:
You left out the "or X contains NaN or infinity" part, I suppose because it is now controlled by `force_all_finite` ? 
I think it might still be useful to at least reference the other keyword, since it is also about "validation"